chain/events/filter/index_migrations.go

```diff
@@ -135,19 +135,11 @@ func migrationVersion2(db *sql.DB, chainStore *store.ChainStore) sqlite.Migratio
 	}
 }
 
-// migrationVersion3 migrates the schema from version 2 to version 3 by creating two indices:
-// 1) an index on the event.emitter_addr column, and 2) an index on the event_entry.key column.
+// migrationVersion3 migrates the schema from version 2 to version 3. It originally added two
+// indices: 1) an index on the event.emitter_addr column, and 2) an index on the event_entry.key
+// column. However, as of version 7, these indices have been removed as they were found to be a
```

Comment:
As per below, the `event_entry.key` Index was removed in migration4. So we should preserve that comment to document history. Only the `event.emitter_addr` Index is being removed in migration 7.